### ISSUE TITLE
"[ERROR] Scanning step _ failed" Not showing all Pokemon

### ISSUE THREAD

**Issue by Liamalia** [AHAAAAAAA/PokemonGo-Map#2632]
I am totally new to programming, just scratching the surface. I set up my map from the YouTube video tutorial linked on this page, so I've been working it strictly from the Python folder command prompt. I've seen a lot of comments mentioning My SQL and changing this variable and that variable but basically...

I'm wondering if there are any tutorials out there for people who aren't as computer literate, like myself, on how to fix the fact that my searches are full of holes and not showing all of the Pokemon. I feel like the first couple of days there were TONS and now my map is literally showing like 2 Pidgey and a Rattata.

TLDR; I suck at programming. I set my map up via YouTube tutorial. Any simple tutorials on how to fix the Pokemon not showing up glitch? Any tutorials on how to run multiple search locations?

**Comment by a-moss** [comment]
Yesterday Niantic implemented a throttling system of 3 requests/second (I believe this is the limit, correct me if wrong) per account. 

This throttle is why you are seeing less pokemon and receiving the error in console.

A few people are working on workarounds for this, however, none of them are stable. If you are feeling adventurous, you can try out the development branch and try using the beehive generator with multiple accounts feature. Otherwise, wait for them to merge it to the master branch.

**Comment by Natglobe** [comment]
Is it possible for the python application to run less than 3 requests/second?

**Comment by Paralysing** [comment]
For this map, yes, just set -t to a number less than 4. (-t 3)

**Comment by Natglobe** [comment]
Paralysing is right. Just add -t 3 at the end of the command.

**Comment by talentedmrripley** [comment]
You can also do -sd 1.1 to get as close to the limit as possible.  Basically anything > 1 should be fine.

**Comment by Paralysing** [comment]
Seems like -sd 1 also works.

**Comment by Liamalia** [comment]
So use -t 3 and -sd 1? Also, any idea why some Pokemon aren't showing up in my search area, but if I move the search area to somewhere a couple hundred feet away (areas are now overlapping) they begin to pop up? For example, there's a truck stop in my area that is a super hot spot and used to pop up a ton of Pokemon at once, but then started only popping up maybe 2 or 3. I moved the search area to a gas station in the same parking lot and more popped up, then to a restaurant in the parking lot and even more popped up. All three areas were overlapping, but it just wasn't catching them on the first or second search.

**Comment by Paralysing** [comment]
-t 3 -sd 1 is the lowest you could go, for stability, I would suggest something higher than -sd 1

**Comment by Paralysing** [comment]
There is also a pr that supports multiple accounts to bypass the throttlie issue, #2568

**Comment by Liamalia** [comment]
Thank you so much!! What is the sd for?

**Comment by Paralysing** [comment]
-sd stands for scan delay, it's used to delay scanning.

**Comment by danidzs** [comment]
Have you found why some Pokemon aren't showing up? I have the same problem

**Comment by QuadTog** [comment]
Yes. The search locations overwrite themselves in the .DB so you can lose Pokemon that spawn in the scanned area overlaps. 

If you find a missing Pokemon you can double check by offsetting your scan so the spawn does not reside within an overlap and check again.

**Comment by Liamalia** [comment]
I'm having the opposite problem though. They're only popping up when I have two or three scan areas overlapped.

**Comment by kylebrower** [comment]
Any way of knowing if the throttling system has been changed since this issue was brought up? My map is not displaying every pokemon like it used to. 

I'm using -t 3 -sd 1.1 still. In the Heroku console I see

2016-07-31T00:11:34.736922+00:00 app[web.1]: 2016-07-31 00:11:34,736 [search_thread-1][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-31T00:11:35.642023+00:00 app[web.1]: 2016-07-31 00:11:35,641 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-31T00:11:35.869196+00:00 app[web.1]: 2016-07-31 00:11:35,868 [search_thread-2][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-31T00:11:35.918391+00:00 app[web.1]: 2016-07-31 00:11:35,917 [search_thread-1][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-31T00:11:36.858817+00:00 app[web.1]: 2016-07-31 00:11:36,858 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-31T00:11:37.079544+00:00 app[web.1]: 2016-07-31 00:11:37,079 [search_thread-2][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-31T00:11:37.126096+00:00 app[web.1]: 2016-07-31 00:11:37,125 [search_thread-1][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-31T00:11:38.072140+00:00 app[web.1]: 2016-07-31 00:11:38,071 [search_thread-0][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-31T00:11:38.290305+00:00 app[web.1]: 2016-07-31 00:11:38,289 [search_thread-2][        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms

Occasionally something will come up but it seems like I am silently getting blocked (aka no pokemon data retrieved). I'm on a up-to-date develop branch at the moment.

**Comment by Paralysing** [comment]
Not sure, I'm using -t 3 -sd 1.2 and my ssh logs look fine.

2016-07-30 20:17:33,253 [        models] [   INFO] Upserted 0 pokemon, 29 pokestops, and 5 gyms
2016-07-30 20:17:33,257 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-30 20:17:33,268 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-30 20:17:33,280 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-30 20:17:33,365 [        models] [   INFO] Upserted 1 pokemon, 26 pokestops, and 5 gyms
2016-07-30 20:17:33,419 [        models] [   INFO] Upserted 0 pokemon, 32 pokestops, and 4 gyms
2016-07-30 20:17:34,421 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-07-30 20:17:34,430 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
